File: ordinacija-fe/src/app/dtos/appointment-create-dto.ts
export class AppointmentCreateDto {
  duration: number;
  date: string;
  time: number;
  jmbg: string;
  email: string;

  constructor(
    duration: number,
    date: string,
    time: number,
    email: string,
    jmbg: string,
  ) {
      this.duration = duration;
      this.date = date;
      this.time = time;
      this.email = email;
      this.jmbg = jmbg;
  }
}

File: ordinacija-be/DTOs/AppointmentCreateDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ordinacija_be.DTOs
{
    public class AppointmentCreateDTO
    {
        public int Duration { get; set; }
        public string Date { get; set; }
        public long Time { get; set; }
        public string JMBG { get; set; }
        public string Email { get; set; }
    }
}
